src/test/java/com/google/firebase/auth/FirebaseAuthIT.java

```diff
@@ -929,6 +986,53 @@ public void onSuccess(ListProviderConfigsPage<SamlProviderConfig> result) {
     assertNull(error.get());
   }
 
+  private UserRecord createUser(
```

Comment:
You can possibly change this into `UserRecord createRandomUser()`. In the body you can use `RandomUser` to generate random data. That's a much simpler signature than passing 3 arguments.